### DISCUSSION TITLE
Not updating phone app since updating TSML

### DISCUSSION THREAD

**Discussion by RBIWebsite** [code4recovery/12-step-meeting-list#478] [Q&A] [answered]
Since updating to V3.11.2  I'm not seeing meeting data transfer to the phone app. Ive edited meeting times and info, and even added a brand new one, and 2 days later the old info is still on the app.

from Readingberksintergroup.org

 **Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by joshreisner** [comment, marked as answer]
hi @RBIWebsite did you report this yet to appsupport@aa.org? there are lots of reasons why a listing might not update in meeting guide, and they are the best equipped to know what's going on

**Reply by tech2serve** [reply]
Hey @RBIWebsite,

Curious if you heard back from App Support?

For the heck of it, I looked at your feed to make sure it is working (@joshreisner had probably already done so) but I wanted to [show it to you](https://readingberksintergroup.org/wp-admin/admin-ajax.php?action=meetings). 

When you have a question about whether or not an update is making it to Meeting Guide, you can inspect your feed to see if the change is present. Assuming it is, then contacting App Support is the only thing to do.

**Comment by RBIWebsite** [comment]
Yes, i have. No response yet.thanks for your time, hopefully they will be able to sort it out.


-------- Original Message --------
Subject: Re: [code4recovery/12-step-meeting-list] Not updating phone app
since updating TSML (#476)
From: Josh Reisner ***@***.***>
Date: Wed, July 28, 2021 6:30 am
To: code4recovery/12-step-meeting-list
***@***.***>
Cc: Steve P ***@***.***>,  Mention
***@***.***>

 hi @RBIWebsite did you report this yet to ***@***.***? there are lots of reasons why a listing might not update in meeting guide, and they are the best equipped to know what's going on —You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by RBIWebsite** [comment]
Thanks for the reply & quick lesson, I appreciate it!
No, I haven’t heard back from app support. One of the changes I made on the website Monday  has made it to the app. One I made Tuesday night still has not. Perhaps it’s constipated………😳😬🤣😂
